src/v2/styles/_emoji-replacement.scss

```diff
@@ -1,5 +1,7 @@
-/* declare a font faces for our Emoji Replacement font, based on the default font used by Stream Chat React */
-$assetsPath: '../../assets' !default;
+@use 'variables';
+
+$assetsPath: variables.$assetsPath !default;
```

Comment:
Avoids breaking change, and lets integrators to keep using the existing sytnax: `@use 'stream-chat-<SDK anme>/../emoji-replacement' with ($assetsPath: '../../../../assets');`